src/QMCWaveFunctions/TrialWaveFunction.h

```diff
@@ -168,11 +168,6 @@ class TrialWaveFunction : public MPIObjectBase
                         ParticleSet::ParticleGradient_t& fixedG,
                         ParticleSet::ParticleLaplacian_t& fixedL);
 
-  /** functions to handle particle-by-particle update
-   * both ratio and full_ratio will be replaced by calcRatio which will handle ValueType.
-   */
-  RealType ratio(ParticleSet& P, int iat);
```

Comment:
This is removed, but the calcRatio documentation references it.  Can you clean this documentation up a bit?

Reply:
This is an oversight.